Content/src/Client/Client.fs

```diff
@@ -138,11 +150,26 @@ let stream model msgs =
 let update (msg : Msg) (currentModel : Model) : Model =
     match currentModel.Counter, msg with
     | Some counter, Increment ->
+    #if (bridge)
+        Bridge.Send (ServerMsg.Increment)
```

Comment:
Combination of `reaction && bridge` are using the side-effects directly instead of proper commands using `Cmd.bridgeSendOr`